conformance_tests/sysman/test_sysman_power/src/test_sysman_power.cpp

```diff
@@ -194,14 +199,19 @@ TEST_F(
       zes_power_sustained_limit_t pSustainedInitial = {};
       zes_power_burst_limit_t pBurstInitial = {};
       zes_power_peak_limit_t pPeakInitial = {};
-      lzt::get_power_limits(p_power_handle, &pSustainedInitial, &pBurstInitial,
+      auto status = lzt::get_power_limits(p_power_handle, &pSustainedInitial, &pBurstInitial,
                             &pPeakInitial);
+      if (status == ZE_RESULT_ERROR_UNSUPPORTED_FEATURE) {
+        continue;
+      } else {
+        EXPECT_EQ(status, ZE_RESULT_SUCCESS);
+      }
       zes_power_sustained_limit_t pSustainedLater = {};
       zes_power_burst_limit_t pBurstLater = {};
       zes_power_peak_limit_t pPeakLater = {};
-      lzt::get_power_limits(p_power_handle, &pSustainedLater, &pBurstLater,
+      status = lzt::get_power_limits(p_power_handle, &pSustainedLater, &pBurstLater,
```

Comment:
no check for UNSUPPORTED feature here ?

Reply:
if its UNSUPPORTED for a power handle we will skip based on the first call itself on line 205

Reply:
added everywhere now to be consistent